src/convertToObject.js

```diff
@@ -6,7 +6,28 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cleanedString = sourceString.trim();
+
+  const declarations = cleanedString.split(';');
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable. Also, consider using `filter` to remove empty declarations resulting from consecutive semicolons.